src/org/labkey/test/pages/core/admin/AllowedFileExtensionAdminPage.java

```diff
@@ -0,0 +1,189 @@
+package org.labkey.test.pages.core.admin;
+
+import org.labkey.test.BootstrapLocators;
+import org.labkey.test.Locator;
+import org.labkey.test.Locators;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.components.html.Input;
+import org.labkey.test.pages.LabKeyPage;
+import org.labkey.test.util.PortalHelper;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class AllowedFileExtensionAdminPage extends LabKeyPage<AllowedFileExtensionAdminPage.ElementCache>
+{
+    public AllowedFileExtensionAdminPage(WebDriver driver)
+    {
+        super(driver);
+    }
+
+    public static AllowedFileExtensionAdminPage beginAt(WebDriverWrapper webDriverWrapper)
+    {
+        webDriverWrapper.beginAt(WebTestHelper.buildURL("admin", "allowList"));
```

Comment:
The allowList page has several flavors, need to specify the `FileExtension` version.
```suggestion
        webDriverWrapper.beginAt(WebTestHelper.buildURL("admin", "allowList", Map.of("type", "FileExtension")));
```

Reply:
Added.